### PR TITLE
docs: Manage OpenID session secrets

### PR DESCRIPTION
**Pull request by Kirdock**
Related PR: https://github.com/keptn/keptn/pull/6689/
Signed-off-by: Klaus Strießnig <k.striessnig@gmail.com>

### CODE REVIEW

File: content/docs/0.13.x/operate/user_management/oauth_secrets/index.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+title: OpenID Session Secrets
+description: Manage session and database encryption secret of your Keptn Bridge.
+weight: 30
+---
+
+In this section, the management of the session and database encryption secret of the Keptn Bridge is explained.
```
Comment (thisthat) [inline content/docs/0.13.x/operate/user_management/oauth_secrets/index.md:7]:
```suggestion
In this section, the management of the session and database encryption secrets of the Keptn Bridge is explained.
```

**Review by thisthat — APPROVED** [review]

File: content/docs/0.13.x/operate/user_management/oauth_secrets/index.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+* To reset the session and the database encryption secret of a Keptn installation, first delete the secret:
+
+    ```console
+    kubectl delete secret -n keptn bridge-oauth
+    ```
+
+* [Create the secrets](./#create-session-and-database-encryption-secret) as explained above.
+
+* Re-start the Keptn Bridge since it requires new secrets:
```
Comment (thisthat) [inline content/docs/0.13.x/operate/user_management/oauth_secrets/index.md:83]:
```suggestion
* Re-start the Keptn Bridge to fetch the new secrets:
```

File: content/docs/0.13.x/operate/user_management/oauth_secrets/index.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
+---
+title: OpenID Session Secrets
+description: Manage session and database encryption secret of your Keptn Bridge.
+weight: 30
+---
+
+In this section, the management of the session and database encryption secrets of the Keptn Bridge is explained.
```
Comment (StackScribe) [inline content/docs/0.13.x/operate/user_management/oauth_secrets/index.md:7]:
```suggestion
This section explains the management of the session and database encryption secrets of the Keptn Bridge.
```

File: content/docs/0.13.x/operate/user_management/oauth_secrets/index.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+- `session secret`: This secret is used to hash the session of the user.
+- `database encryption secret`: This secret is used to encrypt the session in the database.
```
Comment (StackScribe) [inline content/docs/0.13.x/operate/user_management/oauth_secrets/index.md:10]:
```suggestion
- `session secret`: Used to hash the session of the user.
- `database encryption secret`: Used to encrypt the session in the database.
```
"This secret is" in these two items seems like superfluous noise but you may disagree.

File: content/docs/0.13.x/operate/user_management/oauth_secrets/index.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+## Create session and database encryption secret
+
+* To create the session and database encryption secret, use the following commands provided by the operating system:
+
+    <details><summary>on Linux </summary>
+    <p>
+
+    * To generate the secrets, use the following commands and store them into the environment variables `OAUTH_SESSION_SECRET` and `OAUTH_DATABASE_ENCRYPT_SECRET`:
```
Comment (StackScribe) [inline content/docs/0.13.x/operate/user_management/oauth_secrets/index.md:19]:
```suggestion
    * Use the following commands to generate the secrets and store the results into the environment variables `OAUTH_SESSION_SECRET` and `OAUTH_DATABASE_ENCRYPT_SECRET`:
```
The existing syntax says that one is storing the commands in the environment variables and I don't think that is the intent.

**Review by StackScribe — COMMENTED** [review]
lgtm

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
Looks good to me. No blockers in spelling check, but we need to configure the bot exceptions so that the master branch remains green

**Review by oleg-nenashev — APPROVED** [review]
+1 from me since the bot is disabled

**Pull request merged**